app/views/customers/index.html.erb

```diff
@@ -4,7 +4,7 @@
     <h1>Customers</h1>
   </head>
 <p>Click on 'show' next to a customer to add a pet or booking to their account</p>
-<a href="customers/new" class="btn btn-success" role="button">New Customer</a>
+<a href="/customers/new" class="btn btn-success" role="button">New Customer</a>
```

Comment:
Generally this is a pattern we would try to avoid and again comes down to DRY. If we decide to change the path for new customers to `/clients/new` then we would have to change it in lots of places.

Fortunately, the `config/routes.rb` file gives us a lot of benefits with all of those `xyz_path` methods that it magically creates (and you're using below)

Instead of specifying `/customers/new`, consider using the method `new_customer_path`.

Another thing to consider is using the `link_to` helper method here, like you have below.

Routes (and the helper methods) are quite complicated, and if you want to read more about the magic, see https://guides.rubyonrails.org/routing.html

Reply:
Ok. Thanks for the advice. I will make those changes.